subsys/bluetooth/audio/cap_internal.h

```diff
@@ -87,6 +88,14 @@ struct bt_cap_commander_proc_param {
 			struct bt_vocs *vocs;
 		} change_offset;
 #endif /* CONFIG_BT_VCP_VOL_CTLR_VOCS */
+		struct {
+			bt_addr_le_t addr;
+			uint8_t adv_sid;
+			uint32_t broadcast_id;
+			uint16_t pa_interval;
+			uint8_t num_subgroups;
+			struct bt_bap_bass_subgroup *subgroups;
```

Comment:
Subgroups are a pointer here instead of an array, which means that the `bt_cap_commander_broadcast_reception_start_member_param` shall remain valid until the end of the procedure. I don't think that's documented, is it?

Alternative would be to copy the data